biosys/apps/main/tests/api/test_serializers.py

```diff
@@ -0,0 +1,35 @@
+from django.test import TestCase
```

Comment:
Not related test. Testing that the API returns an error for dataset with not unique name.